remote-wallet/src/ledger_error.rs

```diff
@@ -0,0 +1,86 @@
+use num_derive::FromPrimitive;
+use thiserror::Error;
+
+#[derive(Error, Debug, Clone, FromPrimitive, PartialEq)]
+pub enum LedgerError {
+    #[error("Solana app not open on Ledger device")]
+    NoAppResponse = 0x6700,
+
+    #[error("Ledger sdk exception")]
+    SdkException = 0x6801,
+
+    #[error("Ledger invalid parameter")]
+    SdkInvalidParameter = 0x6802,
+
+    #[error("Ledger overflow")]
+    SdkExceptionOverflow = 0x6803,
+
+    #[error("Ledger security exception")]
+    SdkExceptionSecurity = 0x6804,
+
+    #[error("Ledger invalid crc")]
+    SdkInvalidCrc = 0x6805,
+
+    #[error("Ledger invalid checksum")]
+    SdkInvalidChecksum = 0x6806,
+
+    #[error("Ledger invalid counter")]
+    SdkInvalidCounter = 0x6807,
+
+    #[error("Ledger blind signing of messages not supported")]
```

Comment:
@t-nelson, should this be "not enabled" instead of "not supported"?

Reply:
This reply actually isn't restricted to blind-signing.  It's the generic `NOT_SUPPORTED` exception from the Nano S SDK.  I'd relax the message to something like "Ledger operation not supported"

Reply:
So which error tells me "unrecognized format and blind signing is not enabled"?

Reply:
It's just throwing the generic `NOT_SUPPORTED`, ATM.  We can add an explicit one.